test/testprocess.c

```diff
@@ -0,0 +1,177 @@
+#include <SDL3/SDL.h>
+#include <SDL3/SDL_main.h>
+#include <SDL3/SDL_test.h>
+
+#ifdef SDL_PLATFORM_WINDOWS
+#define EXE ".exe"
+#else
+#define EXE ""
+#endif
+
+/*
+ * FIXME: Additional tests:
```

Comment:
Looks like the tests don't fail, so use SDLTest_TestSuiteRunner.

```suggestion
 * FIXME: Additional tests:
 * - use SDLTest_TestSuiteRunner
```